CONTRIBUTING.md

```diff
@@ -129,6 +130,39 @@ We generally review for:
 To speed the process along, we may merge small changes such as spelling and formatting
 into your branch.
 Otherwise, we make suggestions and work with you to finalize the changes.
+<!-- vale Platform.first-person = YES -->
+
+### Review comment style
+
+To make comments clearer, each comment should have an emoji label describing its purpose.
+The following labels are primarily used:
+
+| Emoji | `:code:`              | Meaning      | Can merge  | Description |
+| ----- | --------------------- | ------------ | ---------- | ----------- |
+| 😍    | `:heart_eyes:`        | Praise       | ▶️          | Used when something is well done. No further action required. Should be at least one of these for each review. |
+| ⛏️    | `:pick:`              | Nitpick      | ▶️          | For small details based on preferences. |
+| 🛠️    | `:hammer_and_wrench:` | Suggestion   | 🚫         | To indicate a specific idea for improvement. By default, the suggestion needs to be accepted or otherwise addressed before merging. |
+| 💅    | `:nail_car:`          | Polish       | ▶️          | To indicate a specific idea for improvement that isn't fixing something wrong, but just pointing out ways to improve quality. |
+| 📋    | `:clipboard:`         | To-do        | 🚫         | For small, necessary changes, such as fixing typos. |
+| 🐞    | `:lady_beetle:`       | Issue        | 🚫         | To highlight a specific issue that needs to be fixed. Can be paired with a suggestion if a solution is known. |
+| ❓    | `:question:`          | Question     | 🚫         | For potential concerns that may not be relevant or for areas that aren't completely clear. By default, requires a response. |
+
+The following labels are also possible:
+
+| Emoji | `:code:`              | Meaning      | Can merge  | Description |
+| ----- | --------------------- | ------------ | ---------- | ----------- |
+| 💡    | `:bulb:`              | Thought      | To introduce an idea that came up from reviewing. More general than a suggestion and not focused on the details. Doesn't block merging by default, but can lead to more discussion. |
+| 🧹    | `:broom:`             | Chore        | For small process tasks that need to be done before merging. |
+
+The following decorations can be added for further clarification (to override default of blocking or not):
+
+| Emoji | `:code:`          | Meaning      | Description |
+| ----- | ----------------- | ------------ | ----------- |
+| 🚫    | `:no_entry_sign:` | Blocking     | Used to indicate that the comment blocks merging. |
+| ▶️     | `:arrow_forward:` | Non-blocking | Used to indicate that the comment doesn't block merging. |
```

Comment:
:hammer_and_wrench: I think using the non-ascii character might help to help clarify what's blocking from what isn't.

```suggestion
| ✅     | `:white_check_mark:` | Non-blocking | Used to indicate that the comment doesn't block merging. |
```

Reply:
To me, a check mark looks like it's something that has been resolved and therefore doesn't require any thought at all. I feel like that's different than a non-blocking comment, which can be skipped but isn't necessarily resolved yet. I think there might be something better than an arrow, so if you have other ideas I'd welcome them, but I'd probably avoid a check mark.

I'm not sure I get the non-ASCII part of the comment. The ASCII character would be →, no?

Reply:
My phrasing was confusing, I assume I meant that the emoji version would be better than the triangle.
As propositions:
-  🆗 since it's ok to proceed 
-  ➡️ Because the flow is not disrupted
-  ⏸ since it's not "pausing" the process

Reply:
The triangle _is_ the emoji version, so I'm still not sure what the comment should mean.

The last suggestion is amusing to me since it's the opposite of how I think of it. This isn't pausing the process, so I wouldn't use pause here, but it could make sense for the blocking comment (and is thus less negative there). That way pause = block and play = continue.

Reply:
To clarify, the triangle is the emoji for the play button: https://emojipedia.org/play-button/ GitHub just gives it another name.

Reply:
I like the idea of pause and play 🤔  So if you like it too, we can go with that?